sdk/storage/storage-blob/samples/typescript/basic.ts

```diff
@@ -2,45 +2,12 @@
  Setup: Enter your storage account name and shared key in main()
 */
 
-import {
-  BlobServiceClient,
-  Models,
-  SharedKeyCredential,
-  newPipeline,
-  RawTokenCredential
-} from "../../src"; // Change to "@azure/storage-blob" in your package
+import { BlobServiceClient, Models } from "../../src"; // Change to "@azure/storage-blob" in your package
 
 async function main() {
-  // Enter your storage account name and shared key
-  const account = "";
-  const accountKey = "";
-
-  // Use SharedKeyCredential with storage account and account key
-  const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
-
-  // Use RawTokenCredential with OAuth token.  You can find more
-  // TokenCredential implementations in the @azure/identity library
-  // to use client secrets, certificates, or managed identities for
-  // authentication.
-  const tokenCredential = new RawTokenCredential("token");
-  tokenCredential.token = "renewedToken"; // Renew the token by updating token field of token credential
-
-  // Use AnonymousCredential when url already includes a SAS signature
-  // const anonymousCredential = new AnonymousCredential();
-
-  // Use sharedKeyCredential, tokenCredential or anonymousCredential to create a pipeline
-  const pipeline = newPipeline(sharedKeyCredential);
-
   // Create Blob Service Client from Connection String
```

Comment:
I'd prefer keeping the other code.  They were probably added to help customers. We just want to remove pipeline creation, and passing credential to the constructor instead.

Reply:
The other code is in the other file

Reply:
yeah, but they seems to always duplicate these credential code in all samples so I want to keep them for now.  @XiaoningLiu are these intentional?

Reply:
It made some sense to duplicate when we had only two samples `basic.ts` and `advanced.ts`.
Since we have too many samples now, duplicating feels redundant ?

Reply:
yeah, agree. too many unused code blocks reduce the readability of the real meat. we will not duplicate them in new samples.